cdk/infrastructure/constructs/existing/igvf_dev.py

```diff
@@ -0,0 +1,36 @@
+from constructs import Construct
+
+from shared_infrastructure.igvf_dev.connection import CodeStarConnection
```

Comment:
where is this module, I assume it's what links igvfd CDK to igvf-ui?

Reply:
This is its own repo that allows you to use things that already exist within an AWS account (such as VPCs and Slack connections): https://github.com/igvf-dacc/shared-infrastructure

We're not actually deploying anything here, just making existing things importable, so you don't have to redefined things (e.g. the account environment) for every project.